### ISSUE TITLE
Autocomplete multiple accessibility

### ISSUE THREAD

**Issue by AtiposEiji** [primefaces/primeng#17457]
### Describe the bug

Using Axe DevTools, the autoComplete component with the multiple property has two accessibility issues:

1) ARIA input fields must have an accessible name
    Ensure every ARIA input field has an accessible name
   `<ul role="listbox" ng-reflect-ng-class="[object Object]" tabindex="-1" aria-orientation="horizontal" class="p-autocomplete-multiple- 
   container p-component p-inputtext ng-star-inserted">`

   To solve this problem, you need to fix at least (1) of the following:
     - aria-label attribute does not exist or is empty
     - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
     - Element has no title attribute

2) Interactive controls must not be nested
    Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive 
    technologies.

   `<li role="option" class="p-autocomplete-input-token"><input pautofocus="" role="combobox" aria-autocomplete="list" class="p-element" 
      ng-reflect-ng-class="[object Object]" autocomplete="off" tabindex="0" type="text" id="tags" placeholder="Seleziona" aria- 
      expanded="false"></li>`

    To solve this problem, you need to fix the following:
     Element has focusable descendants

    Related Node
    `<input pautofocus="" role="combobox" aria-autocomplete="list" class="p-element" ng-reflect-ng-class="[object Object]" 
     autocomplete="off" tabindex="0" type="text" id="tags" placeholder="Seleziona" aria-expanded="false">`

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-fj7shuqs

### Environment

macOS Sequoia 15.1.1

### Angular version

19.0.7

### PrimeNG version

v19

### Node version

20

### Steps to reproduce the behavior

1. Open DevTools -> axe DevTools
2. Click on Full Page Scan and wait.

### Expected behavior

It would be preferable if they were accessible

**Comment by Diya-Lal** [comment]
I'd like to add another accessibility violation we've encountered with the AutoComplete component that's related to the issues already described here.

## Additional Issue: **Scrollable region must have keyboard access.**

Axe rule ID : scrollable-region-focusable
WCAG:  WCAG 2 Level A, WCAG 2.1.1, WCAG 2.1.3
Impact: serious

Element location: 

`p-autocomplete-panel`

Element source: 

`<div ng-reflect-ng-class="[object Object]" ng-reflect-ng-style="[object Object]" class="p-autocomplete-panel p-component p-ripple-disabled ng-tns-c3756965041-0" style="max-height: 200px;">`

To solve this violation, you need to...
Fix any of the following:

- Element should have focusable content
- Element should be focusable

**Comment by dausm** [comment]
Can this please be prioritized? 

The structure of the multiselect autocomplete should conform to something closer to this [example](https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/layout-grids/#ex2_label) where the input does not exist inside the same element wrapping the chips
